src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/FeedbackController.cs

```diff
@@ -0,0 +1,38 @@
+#nullable enable
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Requests;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class FeedbackController(ContentDbContext context) : ControllerBase
+{
+    [HttpPost]
+    public async Task<ActionResult> CreateFeedback(
```

Comment:
Would be good to get some integration tests in here. We could also use these to test that the server-side validations (mentioned in another comment) are working correctly.